tools/benchmark-cli/src/main/java/org/logstash/benchmark/cli/util/LsBenchFileUtil.java

```diff
@@ -0,0 +1,22 @@
+package org.logstash.benchmark.cli.util;
+
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+
+public final class LsBenchFileUtil {
+
+    public static void ensureDeleted(final File file) throws IOException {
+        if (file.exists()) {
+            if (file.isDirectory()) {
```

Comment:
@jakelandis this I think is my only point of disagreement :D, depending on what we're gonna build on top of things here, clean FS access with the permissions to delete is paramount.
If we fail here, we should just die otherwise we're just making debugging harder without any benefit?